src/algorithms/edgegradient.jl

```diff
@@ -1,19 +1,20 @@
 """
-    EdgeGradient
+    EdgeGradient <: NeutralLandscapeMaker
+
+    EdgeGradient(; direction = 360rand())
+    EdgeGradient(direction)
 
 This type is used to generate an edge gradient landscape, where values change
 as a bilinear function of the *x* and *y* coordinates. The direction is
 expressed as a floating point value, which will be in *[0,360]*. The inner
 constructor takes the mod of the value passed and 360, so that a value that is
 out of the correct interval will be corrected.
 """
-struct EdgeGradient <: NeutralLandscapeMaker
-    direction::Float64
+@kwdef struct EdgeGradient <: NeutralLandscapeMaker
```

Comment:
I would really rather get rid of `@kwdef` - it doesn't seem the be the "community standard" for constructors.

Reply:
That's largely because Parameters.jl predates it so there isn't really a community standard. I even have another package that does this differently! (FieldDefaults.jl). 

Out options using Base are to use `@kwdef` or to write out the constructors - are you saying you would prefer them fully written out?